service/key/key_test.go

```diff
@@ -0,0 +1,86 @@
+package key
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+)
+
+// Test_Key_CAPath tests the CAPath function.
+func Test_Key_CAPath(t *testing.T) {
+	tests := []struct {
+		groupName    string
+		expectedPath string
+	}{
+		{
+			groupName:    "xa5ly",
+			expectedPath: "/certs/xa5ly/ca.pem",
+		},
+	}
+
+	for index, test := range tests {
+		path := CAPath(test.groupName)
+
+		if !reflect.DeepEqual(test.expectedPath, path) {
+			t.Fatalf(
+				"%v: expected path does not match returned path\nexpected: %v\nreturned: %v\n",
```

Comment:
Please use proper format string placeholders. The index is a number and the rest probably strings.

Reply:
Done